cmd/sfncli/runner.go

```diff
@@ -65,12 +67,12 @@ func (t *TaskRunner) Process(ctx context.Context, args []string, input string) e
 		return t.sendTaskFailure(TaskFailureTaskInputNotJSON{input: input})
 	}
 
-	// convention: if the input contains _EXECUTION_NAME, pass it to the environment of the command
-	var executionName *string
-	if e, ok := taskInput["_EXECUTION_NAME"].(string); ok {
-		executionName = &e
-		t.logger.AddContext("execution_name", *executionName)
+	// convention: pass _EXECUTION_NAME into the environment of the command
+	executionName, ok := taskInput["_EXECUTION_NAME"].(string)
+	if !ok {
+		return t.sendTaskFailure(TaskFailureTaskInputMissingExecutionName{input: input})
 	}
+	t.logger.AddContext("execution_name", executionName)
```

Comment:
nit: `execution-name`

Reply:
Leaving this as is because it was with an underscore before